example/main.tf

```
@@ -20,6 +20,7 @@ module "minio" {
   data_dir                        = "/minio/data"
   container_environment_variables = ["SOME_VAR_N1=some-value"]
   use_host_volume                 = true
+  use_canary                      = false
```

Set on the full set of optional features in the example.
```suggestion
  use_canary                      = true
```